Gemfile

```diff
@@ -60,8 +60,8 @@ gem 'pg'
 # gem 'debugger', group: [:development, :test]
 
 
-gem "blacklight", '~> 5.12.1'
-gem "geoblacklight", '~> 0.11.0'
+gem "blacklight"
+gem "geoblacklight", '~> 0.12.1'
```

Comment:
is there a reason we include blacklight here? doesn't geoblacklight already require blacklight?